libs/htmlToText.js

```diff
@@ -0,0 +1,266 @@
+const htmlToText = (html, options = {}) => {
```

Comment:
Do we need this now since we only have the WebView reader?

Reply:
Yeah, I will remove it